azure/image/files/etc/systemd/system/go.mod

```diff
@@ -0,0 +1,3 @@
+// Empty go.mod file to exclude the run-kata\x2dcontainers.mount file
+// from the cloud-api-adaptor Go module when cloud-api-adaptor is
+// imported by other Go projects
```

Comment:
new line missing. Can you please add it?

Reply:
Thanks for the review @knrt10. I've done that change.